### PR TITLE
fix: add note about email_verified OIDC token field for KOP

### PR DESCRIPTION
**Pull request by mhrabovcin**
## Jira Ticket

JIRA: https://jira.d2iq.com/browse/D2IQ-88868

## Description of changes being made
In the https://jira.d2iq.com/browse/COPS-7262 we've discovered that OIDC ID token must have `email_verified` set to value `true` in order to be verified by `kube-oidc-proxy`. This change documents this finding.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4445.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/security/distributed-authnz/index.md
Comment (aweris) [inline pages/dkp/kommander/2.2/security/distributed-authnz/index.md:20]: `[`kube-oidc-proxy`][kube_oidc_proxy]` is this intentional?

File: pages/dkp/kommander/2.0/security/distributed-authnz/index.md
Comment (palexster) [inline pages/dkp/kommander/2.0/security/distributed-authnz/index.md:20]: it's unclear to me what the "it" in the second sentence is referring to, perhaps the field?

File: pages/dkp/kommander/2.2/security/distributed-authnz/index.md
Comment (mhrabovcin) [reply pages/dkp/kommander/2.2/security/distributed-authnz/index.md:20]: Yes, the second part is a link that is pointing to the kube-oidc-proxy project.

**Review by aweris — APPROVED** [review]

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/2.0/security/distributed-authnz/index.md
Comment (mhrabovcin) [reply pages/dkp/kommander/2.0/security/distributed-authnz/index.md]: @cneth I like the proposed change. I'd like to confirm that this wording

```
This library requires email claims to be verified,
```

matches the go library behavior which is, if the `email_verified` field is not present, the token will be verified. If the `email_verified` is present and its value is `true` the token is verified. And if the `email_verified` is present and its value is `false` then the token is rejected.

**Review by gvasquezvargas — APPROVED** [review]
Thanks Martin! Waiting on a second approval to merge.

**Pull request merged**
